theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx

```diff
@@ -0,0 +1,508 @@
+            // ...
+        }
+        ```
+        </TabItem>
+        <!-- prettier-ignore-end -->
+    </Tabs>
+
+![](../../../assets/img/android-getting-started-21_2.png)
+
+In the `AndroidManifest.xml` we can disable the default orientation behavior on our `MainActivity` and lock it in portrait. Then, because we enabled fullscreen orientation coupling with the `setFullScreenOrientationCoupled(true)` method earlier, THEOplayer will take care of the support of seamless landscape activity transition at rotation.
+
+```xml
+<activity android:name=".MainActivity"
+    android:screenOrientation="userPortrait"
+    android:configChanges="orientation|screenSize" >
+    <intent-filter>
+        <action android:name="android.intent.action.MAIN" />
+
+        <category android:name="android.intent.category.LAUNCHER" />
+    </intent-filter>
+</activity>
+```
+
+## Couple THEOplayer with the lifecycle callbacks
+
+In order for THEOplayer to work properly and in sync with the [Activity lifecycle changes](https://developer.android.com/guide/components/activities/activity-lifecycle) (e.g. when we rotate the device or start a new Activity or the app goes to background).
+We need to call the `onResume`, `onPause` and `onDestroy` methods of the `THEOplayerView` when the matching methods are called in the Activity.
+
+<Tabs queryString="lang">
+       <!-- prettier-ignore-start -->
+       <TabItem value="kotlin" label="Kotlin">
+        ```kotlin
+        class MainActivity : ComponentActivity() {
+
+            // ...
+
+            override fun onPause() {
+                super.onPause()
+                theoPlayerView.onPause()
+            }
+
+            override fun onResume() {
+                super.onResume()
+                theoPlayerView.onResume()
+            }
+
+            override fun onDestroy() {
+                super.onDestroy()
+                theoPlayerView.onDestroy()
+            }
+
+            // ...
+
+        }
+        ```
+        </TabItem>
+        <TabItem value="java" label="Java">
+        ```java
+        public class MainActivity extends AppCompatActivity{
+
+            // ...
+
+            @Override
+            protected void onPause() {
+                super.onPause();
+                theoPlayerView.onPause();
+            }
+
+            @Override
+            protected void onResume() {
+                super.onResume();
+                theoPlayerView.onResume();
+            }
+
+            @Override
+            protected void onDestroy() {
+                super.onDestroy();
+                theoPlayerView.onDestroy();
+            }
+
+            // ...
+        }
+        ```
+        </TabItem>
+        <!-- prettier-ignore-end -->
+    </Tabs>
+
+If you would like to customize the Fullscreen Activity, you can take a look at our sample [here](https://github.com/THEOplayer/samples-android-sdk/tree/master/full-screen-handling).
```

Comment:
This is part of the fullscreen section above.

Reply:
Moved